inductiva/_cli/cmd_tasks/kill.py

```diff
@@ -0,0 +1,22 @@
+"""Kills a tasks by id via CLI."""
+
+import inductiva
+
+
+def kill_task(args):
+    """Kills a task by id."""
+    inductiva.tasks.Task(args.id).kill(wait_timeout=args.wait)
```

Comment:
This needs to be `args.wait_timeout` since we changed the argument below.

Reply:
Changed. I algo added 
"> Or via the CLI by calling inductiva tasks -id <task id>."
to the translations file.